benchmarks/llama3.1_v6e-256_benchmarks.py

```diff
@@ -0,0 +1,44 @@
+from maxstar_model_configs import llama2_70b_4096
+from maxtext_xpk_runner import BenchmarkRunner
+from maxtext_xpk_runner import HWConfig
+from maxtext_xpk_runner import SWconfig
+from maxtext_xpk_runner import xpk_benchmark_runner
+from maxtext_xpk_runner import XpkConfig
+
+
+DATE = '20241009'
+BASE_DOCKER_IMAGE = 'maxtext_base_image'
+
+ZONE = 'europe-west4'
+PROJECT = 'tpu-prod-env-multipod'
+CLUSTER_NAME = 'mlperf-v6e-256'
+DEVICE_TYPE = 'v6e-256'
+NUM_SLICES = 1
+
+v6e_env_configs = SWconfig(
+    base_docker_image=BASE_DOCKER_IMAGE, libtpu_version=DATE
+)
+v6e_256_configs = HWConfig(num_slices=NUM_SLICES, device_type=DEVICE_TYPE)
+
+## todo: replace with llama3.1 model tests
+llama2_70b_4096 = BenchmarkRunner(
```

Comment:
Nice, we can even use argparser to take in the model name argument. The benefit of this is we can have one common benchmark_runner.py file. Curious what you think of this approach?

Reply:
Thanks for the suggestion, I think it's a good idea, pls find the updated code.